ui/src/api/index.js

```diff
@@ -12,5 +12,8 @@ export { patron } from './patrons/patron';
 export { series } from './series/series';
 export { stats, circulationStats } from './stats';
 export { vocabulary } from './vocabularies/vocabulary';
-export { order, vendor } from './acquisition';
-export { borrowingRequest, library } from './ill';
+export { acqOrder, acqVendor } from './acquisition';
+export {
```

Comment:
added this rename just to be able to split in 2 PRs. It is remove in the next PR.